src/View/Admin/admin.html.twig

```diff
@@ -8,14 +8,14 @@
 		<h1 id="dashboard_title">bandes dessinées</h1>
 		<div class="row justify-content-center table-responsive">
 			<div class="col-8">
-				<button id="add_button" type="button" class="btn btn-dark">Ajouter</button>
+				<a href"admin/add"><button href"/admin/add" id="add_button" type="button" class="btn btn-dark">Ajouter</button></a>
```

Comment:
href is not a valid attribute of button tag

Reply:
Changes done